releases/release-1.31/release-notes/maps/pr-122812-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 122812
+releasenote:
+  text: The iptables mode of kube-proxy now tracks packets that are wrongfully marked
```

```suggestion
  text: Added support for kube-proxy iptables mode to track packets that were wrongfully marked invalid by conntrack and subsequently dropped by introducing `kubeproxy_iptables_ct_state_invalid_dropped_packets_total` metric
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653166296